example/models_directory/configuration_interaction.json

```diff
@@ -0,0 +1,28 @@
+{
+    "tier1": "physics-based",
+    "tier2": {
+        "name": "quantum-mechanical",
+        "slug": "quantummechanical"
```

Comment:
Maybe "qm" for brevity?

Reply:
Done in https://github.com/Exabyte-io/esse/pull/139/files#diff-e9574187048c782764567d74ba0fdcdeR8